amethyst_network/src/filter.rs

```diff
@@ -4,7 +4,7 @@ use std::net::SocketAddr;
 use std::marker::PhantomData;
 
 /// Network filter base trait providing an event filtering interface.
-pub trait NetFilter<T> where T: Send+Sync+PartialEq{
+pub trait NetFilter<T>: Send+Sync where T: Send+Sync+PartialEq{
```

Comment:
You see here how bounds on types are spreading and you can' stop it.

Reply:
It didn't spread too far actually after my refactor.